crates/resvg/tests/integration/main.rs

```diff
@@ -85,7 +85,7 @@ pub fn render_inner(name: &str, test_mode: TestMode) -> usize {
                 size.width() as f32 / tree.size().width() as f32,
                 size.height() as f32 / tree.size().height() as f32,
             );
-            resvg::render(&tree, render_ts, &mut pixmap.as_mut());
+            resvg::render(&tree, render_ts, &mut pixmap.as_mut()).expect("Failed to render");
```

Comment:
The best would be to resturn a `Option<usize>` or a `Result<usize, String>`

I don't really like to `.expect()` but I do it now for this MR for no breaking change